org.bluesabre.MenuLibre.metainfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.bluesabre.MenuLibre</id>
+
+  <name>MenuLibre</name>
+  <summary>MenuLibre is an advanced FreeDesktop.org compliant menu editor</summary>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+
+  <description>
+    <p>All fields specified in the FreeDesktop.org Desktop Entry and Menu specifications are available to quickly update. Additionally, MenuLibre provides an editor for the launcher actions used by applications such as Unity and Plank.</p>
+    <ul>
+      <li>A beautiful interface powered by the latest version of GTK+</li>
+      <li>Create new launchers, or modify existing ones with complete control over common settings and access to advanced settings</li>
+      <li>Add, remove, and adjust desktop actions: powerful shortcuts available used by Unity, Xfce, and Pantheon</li>
+      <li>Easily rearrange menu items to suit your needs</li>
+    </ul>
+  </description>
+
+  <launchable type="desktop-id">org.bluesabre.MenuLibre.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/bluesabre/menulibre/wiki-assets/readme/menulibre-github.png</image>
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release version="2.2.3" date="2021-09-26"></release>
+  </releases>
+</component>
```

Comment:
upstream already has a metainfo

Reply:
Huh yeah, I completely forgot about it. Thanks.